examples/blockly-svelte/src/components/BlocklyComponent.svelte

```diff
@@ -1,6 +1,6 @@
 <script>
   import { onMount } from 'svelte';
-  import Blockly from 'blockly';
+  import * as Blockly from 'blockly';
```

Comment:
This should import each thing it needs individually as specified [here](https://developers.google.com/blockly/guides/get-started/get-the-code#imports). If you don't want to fix that now, feel free to file a bug to follow up.

Here and in Vue.

Reply:
I don't need the generators, but I do need to have core, library blocks, and english langfiles.

I updated the imports to 
```
  import * as Blockly from 'blockly/core';
  import * as libraryBlocks from 'blockly/blocks';
  import * as En from 'blockly/msg/en';
  ```
and added a call to `Blockly.setLocale(En)`. This (and the `inject`) call fail because `Blockly` is `undefined`.

Reply:
I applied the same change to the Vue example and it worked fine. Do you know what the difference is? @cpcallen to comment as well.

Reply:
The Vue demo is built using Vite. The Svelte demo is built using rollup. Gonna guess some of the packaging changes aren't compatible with our rollup config, but our config is also pretty out of date (e.g. we're using some commonjs plugin whose git repo was archived 3 years ago) so who knows where the problem lies. This probably warrants Christopher's eyes.

I would probably start by trying a new rollup config from scratch if they have an updated hello world example.

I'll also note that rollup says it is the bundler behind Vite, so the fact that it works over there seems promising. More likely to point to a problem in our specific rollup config than a deeper incompatibility

Reply:
Fixed by updating the version of rollup and some of their plugins. Refreshingly, everything that needed to be replaced or updated was clearly labeled.